### PR TITLE
VideoToSequence sample

### PR DESCRIPTION
**Pull request by rrawther**
Initial implementation for VideoToSequence sample
Seeking and decoding has some issues and needs to fix

### CODE REVIEW

File: utils/rocvideodecode/roc_video_dec.cpp [hunk 1]
```diff
@@ -431,13 +431,14 @@ int RocVideoDecoder::HandleVideoSequence(RocdecVideoFormat *p_video_format) {
  * @return true : success
  * @return false : fail
  */
-bool RocVideoDecoder::SetReconfigParams(ReconfigParams *p_reconfig_params) {
+bool RocVideoDecoder::SetReconfigParams(ReconfigParams *p_reconfig_params, bool b_force_recofig_flush) {
```
Comment (LakshmiKumar23) [inline utils/rocvideodecode/roc_video_dec.cpp]: Typo in `b_force_reconfig_flush`

File: samples/videoToSequence/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,101 @@
+################################################################################
+# Copyright (c) 2023 - 2024 Advanced Micro Devices, Inc.
```
Comment (LakshmiKumar23) [inline samples/videoToSequence/CMakeLists.txt]: Copyright from 2024

File: samples/videoToSequence/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```shell
+mkdir video_decode_sample && cd video_decode_sample
+cmake ../
+make -j
+```
+
+## Run
+
+```shell
+./videodecode -i <Input file/folder Path [required]>
```
Comment (LakshmiKumar23) [inline samples/videoToSequence/README.md]: change to `./videotosequence` if that's the executable name

File: samples/videoToSequence/videotosequence.cpp [hunk 4]
```diff
@@ -0,0 +1,507 @@
+/*
+Copyright (c) 2023 - 2024 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (LakshmiKumar23) [inline samples/videoToSequence/videotosequence.cpp]: copyright from 2024

**Review by LakshmiKumar23 — CHANGES_REQUESTED** [review]

File: samples/videoToSequence/videotosequence.cpp [hunk 5]
```diff
@@ -0,0 +1,504 @@
...
+            }
+            continue;
+        }
+        if (!strcmp(argv[i], "-o")) {
+            if (++i == argc) {
+                ShowHelpAndExit("-o");
+            }
+            output_folder_path = argv[i];
+            if (!output_folder_path.empty()) {
+#if __cplusplus >= 201703L && __has_include(<filesystem>)
```
Comment (LakshmiKumar23) [inline samples/videoToSequence/videotosequence.cpp:282]: @rrawther this is already added at he beginning of the file. We should not redefine. Please check

**Review by LakshmiKumar23 — APPROVED** [review]

**Review by AryanSalmanpour — APPROVED** [review]

**Pull request merged**